src/js/index.js

```diff
@@ -50,11 +51,11 @@ function handleTags() {
 
 	if (tags.length === 0) {
 		history.pushState(null, '', window.location.pathname);
-		return;
 	}
 
 	Array.from(contentEl.querySelectorAll('.item')).forEach(el => {
-		const show = tags.reduce((sho, tag) => sho || el.classList.contains(`tag-${tag}`), false);
+		const show = (showExpired.checked || !el.classList.contains('tag-expired')) &&
```

Comment:
```suggestion
		const show = (showExpired.checked || !el.classList.contains('tag-expired')) &&
	(tags.reduce((sho, tag) => sho || el.classList.contains(`tag-${tag}`), false));
```

Again with the multiline suggestion, but 

 - No need to check if the tag is `expired`, because we filter it out in the gulpfile
 - No need to check if the taglist is empty, because if it is, we would have returned previously

Reply:
Sorry for the significant delay, I forgot to commit & push this 😅